pkg/api/device_req_resp.go

```diff
@@ -0,0 +1,35 @@
+package api
+
+import "github.com/pborman/uuid"
+
+const (
+	// DeviceEnabled represents enabled
+	DeviceEnabled = "Enabled"
+
+	// DeviceFrozen represents frozen
+	DeviceFrozen = "Frozen"
+
+	// DeviceEvacuated represents evacuated
+	DeviceEvacuated = "Evacuated"
```

Comment:
Please name this as `DeviceDisabled` which is simple and intuitive.

Also, I'll need some more detail in code comments about the states. For example, what does it mean for a device to be "frozen" ?